src/odemis/util/__init__.py

```diff
@@ -110,24 +110,33 @@ def almost_equal(a, b, atol=1e-18, rtol=1e-7):
     return math.isclose(a, b, rel_tol=rtol, abs_tol=atol)
 
 
-def rot_almost_equal(a, b, atol=1e-18, rtol=1e-7):
+def rot_almost_equal(a: float, b: float, atol: float=1e-18, rtol: float=1e-7) -> bool:
     """
     Check the rotation difference between two radian angles is within a margin
-    a (float) an angle, in radians
-    b (float) an angle, in radians
-    atol (float): absolute tolerance
-    rtol (float): relative tolerance
-    returns (bool): True if a and b rotation is within a margin
+    a: an angle, in radians
+    b: an angle, in radians
+    atol: absolute tolerance
+    rtol: relative tolerance
+    returns: True if a and b rotation is within a margin
     """
     if a == b:
         return True
 
-    tau = 2 * math.pi
-    tol = max(atol, max(abs(a % tau), abs(b % tau)) * rtol)
-    if abs(((a - b) + math.pi) % tau - math.pi) <= tol:
-        return True
+    # Convert rtol to an absolute value (based on the largest argument)
+    # and pick the final tolerance as the biggest of the tolerances.
+    tol = max(atol, abs(rot_mod_mpi_ppi(a)) * rtol, abs(rot_mod_mpi_ppi(b)) * rtol)
+    # calculate the difference as a value between -pi and pi, and check it's near 0
+    return abs(rot_mod_mpi_ppi(a - b)) <= tol
 
-    return False
+
+def rot_mod_mpi_ppi(a: float) -> float:
```

Comment:
Indeed, typically it's nicer to have a verb as the first word of a function. I used "rot" instead of "angle" because that was already what we used in other functions. What do you think of `map_rot_to_mpi_ppi`? Or is that too confused with a rotten pie?

Reply:
`map_rot_to_mpi_ppi` is good!

Reply:
I've looked online, and we're not the first one to create such function. Who would have thought!
After reviewing some names, I'd suggest `wrap_to_mpi_ppi()`. What do you think?

Reply:
sounds good!